src/AddTodoForm.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const AddTodoForm = () => {
+  return (
+    <>
+      <form>
+        <label htmlFor="todoTitle">Title: </label>
+        <input id="todoTitle" type="text" name="title" />
```

Comment:
Good use of semantic html and accessibility when using the htmlFor attribute.

Reply:
@gsvidal Thank you!